<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>主窗口</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>1</y>
     <width>771</width>
     <height>591</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>路径</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="txtFolder"/>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="btnChoose1">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>24</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Images/search.png</normaloff>:/Images/search.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="btnRead1">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>读取</string>
      </property>
     </widget>
    </item>
    <item row="0" column="5" rowspan="2">
     <widget class="QPushButton" name="btnSetting">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Images/setting.png</normaloff>:/Images/setting.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="txtExport"/>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="btnChoose2">
      <property name="minimumSize">
       <size>
        <width>30</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>30</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Images/search.png</normaloff>:/Images/search.png</iconset>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="btnRead2">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>导出</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>过滤</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="txtFilter"/>
    </item>
    <item row="2" column="2">
     <widget class="QCheckBox" name="cbSubFolder">
      <property name="text">
       <string>包含子文件夹</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="btnNew">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>新建</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="6">
     <widget class="QTableView" name="tableView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
